### PR TITLE
[aarch64] build cuda 12.6 manywheel dockers

### PR DESCRIPTION
**Pull request by tinglvv**
Add Builds sbsa 12.6 manywheel dockers to workflow
Related to #138440 
cc @atalman

### CODE REVIEW

**Comment by tinglvv** [comment]
@atalman could you help create the tag cuda12.6 under https://hub.docker.com/r/pytorch/manylinuxaarch64-builder as well? Thanks.

File: .github/workflows/build-manywheel-images.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
     runs-on: "${{ needs.get-label-type.outputs.label-type }}linux.9xlarge.ephemeral"
     strategy:
       matrix:
-        cuda_version: ["12.6", "12.4", "12.1", "11.8"]
+        cuda_version: ["12.4", "12.1", "11.8"]
```
Comment (atalman) [inline .github/workflows/build-manywheel-images.yml:97]:
```suggestion
        cuda_version: ["12.6", "12.4", "12.1", "11.8"]
```

Comment (atalman) [reply .github/workflows/build-manywheel-images.yml:97]: I believe w need to keep 12.6 here

**Review by atalman — APPROVED** [review]
lgtm. We can merge on green

**Comment by tinglvv** [comment]
Trunk failure reads runner issue, should be irrelevant, merging
https://github.com/pytorch/pytorch/actions/runs/12208726882/job/34068119592
`"Please file an issue on pytorch/pytorch reporting the faulty runner. Include a link to the runner logs so the runner can be identified"[0m
2024-12-07T07:53:18.8539005Z [36;1mif [[ $ngpu -eq 0 ]]; then[0m
2024-12-07T07:53:18.8539323Z [36;1m    echo "Error: Failed to detect any GPUs on the runner"[0m`

**Pull request closed without merging**